epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModel.java

```diff
@@ -137,7 +183,54 @@ public void bind(T view, EpoxyModel<?> previouslyBoundModel) {
    *
    * @see EpoxyAdapter#onViewRecycled(EpoxyViewHolder)
    */
-  public void unbind(T view) {
+  public void unbind(@NonNull T view) {
+  }
+
+  /**
+   * Called when the model has changed but still represents the same item. Return true if you want
+   * to animate these changes manually in the viewholder instead of simple cross fading or whatever
+   * animation the {@link android.support.v7.widget.RecyclerView.ItemAnimator) does by default.
+   *
+   * @return true if you wish to animate changes in the model
+   */
+  public boolean canReuseViewHolder(@NonNull EpoxyModel<?> previouslyBoundModel) {
```

Comment:
I'm not yet comfortable using `canAnimateChanges` as a proxy for whether to reuse the viewholder.

A common case I have is where a view has a nested image, and that image changes. In this case everything else is the same and we just want to load the new image. We really don't want to create a new viewholder, because the view is expensive to create and we don't need the cross fade animation. Instead we use Glide to manage our image loading and it automatically uses our placeholder, and crossfades the new image as it is loaded. however, it should not need to implement `bindChangesAnimated` and return an animation.

That is a case we need to support. To do that, we need

1. a `bind` method that passes the previous model
2. A way to set `canReuseViewHolder` to true
3. No animation returned from `bindChangesAnimated`